components/docs-chef-io/content/habitat/reusable/md/license_key.md

```diff
@@ -0,0 +1,13 @@
+## Add a License Key
+
+In order to download and sync official Chef-maintained packages from the Public Builder to your On-Prem Builder instance, a valid license key is required.
+
+### Step 1: Obtain Your License Key
+
+1. Log into your customer portal.
+2. Copy the license key linked to your asset.
+3. Enter the license key in Profile Page of Public Builder.
+
+### Step 2: Add the License Key to Your On-Prem Account
+
+You can enter your license key just below the Personal Access Token text field in order for your account to access official chef maintained packages.
```

Comment:
Is this a new field below the current Personal Access Token field? Does it have a label?

This is what I see in the Builder SaaS:

![image](https://github.com/user-attachments/assets/f8aa7a55-bda2-4c52-8a23-cecba64b4aa4)